src/main/webapp/css/style.css

```diff
@@ -65,6 +65,10 @@
 
 .gameButton {
     margin: none;
+    box-shadow: 0 0 1px 0
+        inset, 0 0 0 1px;
```

Comment:
I would rather this be a shadow that matches the color of the gameButton of some sort. I think the white border looks a little harsh.

Reply:
Or something similar. I am not sure.

Reply:
I was not too sure what to do, so I took out the shadow

Reply:
I just fixed it using the same code from the palette!